assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -21,7 +21,7 @@ module "bootkube" {
   # Disable the self hosted kubelet.
   disable_self_hosted_kubelet = var.disable_self_hosted_kubelet
 
-  bootstrap_tokens     = var.enable_tls_bootstrap ? [local.controller_bootstrap_token, local.worker_bootstrap_token] : []
+  bootstrap_tokens     = var.enable_tls_bootstrap ? concat(module.controller.*.bootstrap_token, module.worker.*.bootstrap_token) : []
```

Comment:
Workers module do not support non-tls-bootstrap config, so `var.enable_tls_bootstrap` needs to be removed.

For clean git history, I'd remove it first, then migrate to module.